htmd/parameterization/ffmolecule.py

```diff
@@ -126,33 +126,30 @@ def _rename_mol(self):
         # This fixes up the atom naming and reside name to be consistent
         # NB this scheme matches what MATCH does. Don't change it
         # Or the naming will be inconsistent with the RTF
-        import re
 
-        hh = dict()
+        sufices = dict()
 
+        print('\nRename atoms:')
         for i in range(len(self.name)):
-            # This fixes the specific case where a name is 3 characters, as X-TOOL seems to make
-            t = self.name[i].upper()
-            if re.match( "^[A-Z][A-Z][A-Z]", t ): 
-               t = t[0]
-            # Remove any character that isn't alpha
-            t = re.sub('[^A-Z]*', "", t )
-            if t != self.name[i].upper():
-#               print(" Atom #%d renamed from [%s] to [%s]" %(i, self.name[i], t ) )
-               pass
+            name = self.name[i].upper()
+
+            # This fixes the specific case where a name is 3 or 4 characters, as X-TOOL seems to make
+            if re.match('^[A-Z]{3,4}$', name):
+               name = name[:-2] # Remove the last 2 characters
```

Comment:
Last question. Sorry for my lack of knowledge of the specific problem here, but do you really want to remove the last 2 characters? So if it's 4 characters you keep 2 and if it's 3 characters you keep 1? Or you want just the first character?

Reply:
Yes, this is a convention of X-Tool, where the atom name is: element symbol + 2 character. So I want to get rid of these 2 characters. :)

Reply:
ok fine